### DISCUSSION TITLE
Hyper as part of Keypad?

### DISCUSSION THREAD

**Discussion by snowiow** [meow-edit/meow#279] [Q&A]
Hi,
is there a possibility to access `H-{something}` keybindings in Keypad mode as well? I can't find a good key for hyper and was thinking that this would be pretty awesome way to get around it if it's possible, but I couldn't find any info about it.

Thanks in advance 🙂

**Comment by Linerre** [comment]
Hi @snowiow , if I understand your point correctly:

     (meow-leader-define-key
       ;; some other stuff go here
       '("H l" . display-line-numbers-mode))

Then in a buffer, press `SPC` to enter keypad mode, you should see sth like `H -> +prefix` in the cheat sheet. In the example above, `SPC H l` will enable line numbers mode in the current buffer.

**Reply by snowiow** [reply]
Hi, thanks for your answer but not directly. 
I try to give some more context for my problem: I am using the colemak keyboard layout and `n` is the key to go to next line in a normal meow editing buffer. Now I'm using this major modes [kubel](https://github.com/abrochard/kubel) where `n` is bound to list all kubernetes namspaces. But to keep the normal navigation keys I've set the following:

```emacs-lisp
(meow-motion-overwrite-define-key '("n" . next-line)
```
This function overrides `n` to next-line and whatever was set to `n` before to H-n. Meaning press [hyper](http://xahlee.info/emacs/emacs/emacs_hyper_super_keys.html) and n the same time. Because I don't have any leftover keys on my keyboard, I can't bind anything physical to have a hyper key and what I was thinking is: could it somehow be possible to access everything that is bound under hyper like it is for meta (SPC-m) or control (SPC-{most other keys})

**Comment by eshrh** [comment]
WRT your reply above, i don't this is currently easily possible. The C- M- C-M and literal modes are more or less hard coded into the keypad via long cond statements. As far as i know, it would take a fairly significant refactor to get this to work...

**Reply by snowiow** [reply]
ah ok, yes I had a look and you are right. But would support for hyper (and maybe also super) be a feature with which keypad mode could be extended in general or is this rather not wanted? Maybe I would look into it then, if I find some time 🙂

**Reply by DogLooksGood** [reply]
I didn't implement Hyper support because as far as I know, there's no built-in shortcuts with Hyper. Keypad is used to simulate the built-in(or customized but following the key binding convension) key bindings.

**Reply by snowiow** [reply]
ok, I have overwritten `meow-motion-remap-prefix` now, so that I don't need to have a hyper key.
